### DISCUSSION TITLE
Manual/webgl-texture-compression

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#516] [Notes]
# Manual/webgl-texture-compression

https://docs.unity3d.com/Manual/webgl-texture-compression.html

**Comment by radiatoryang** [comment]
Note that the example editor script here (at least as of December 2022) is broken. You have to set the WebGLTextureSubtarget with an enum hack: `buildOptions.subtarget = (int)WebGLTextureSubtarget.ASTC;`
